src/tests/ftest/util/dfuse_utils.py

```diff
@@ -410,3 +410,47 @@ def stop(self):
 
         else:
             self.log.info("No hosts running dfuse - nothing to stop")
+
+
+def get_dfuse(test, hosts):
```

Comment:
Didn't I make a change similar to this function myself recently?

Reply:
Yes. You added `DfuseTestBase.load_dfuse`, which initializes `self.dfuse`. This new function initializes a dfuse instance independently of `self.dfuse` so tests can instantiate multiple instances